Tasks/Popko/Task-1/deposit.py

```diff
@@ -0,0 +1,29 @@
+x = input ('Please enter your name:\n')
+print ('Hello,', x, end = '!\n')
+answer = input('Would you like to calculate deposit (y/n)?\n')
+if answer == 'y':
+    p = int(input('Enter the deposit amount in BYN\n'))
+    if p<=0:
+        print ('Incorrect')
+    r = int(input('Enter the deposit interest rate in %\n'))
+    if r<=0:
+        print ('Incorrect')
+    t = int(input('Enter the deposit term in years\n'))
+    if t<=0:
+        print ('Incorrect')
```

Comment:
By adapting these values for the int type and using them in the math expression you get the float number as answer, which tends to accumulate some error, it's better to use decimal for money-related operations, -10 points for the logical mistake